gap/examples.gi

```diff
@@ -714,3 +714,41 @@ function(filt, m, n)
   SetIsSymmetricDigraph(D, true);
   return D;
 end);
+
+InstallMethod(BookDigraphCons,
+  "for IsMutableDigraph and one positive integer",
+  [IsMutableDigraph, IsPosInt],
+  function(filt, m)
+    local pages, book, graph;
+    # special cases
+    if m = 1 then
+      # "the" square graph
+      return CompleteBipartiteDigraph(filt, 2, 2);
+    elif m = 2 then
+      # domino graph
+      return SquareGridGraph(filt, 2, 3);
+    fi;
+    pages := StarDigraph(filt, m);
```

Comment:
In my opinion (just my preference), for the purposes of maximising clarity, since `filt` should always equal `IsMutableDigraph` in this method, then `IsMutableDigraph` should be used directly, rather than `filt`.